LICENSE_SHORT

```diff
@@ -0,0 +1 @@
+Copyright (c) 2022 Airbyte, Inc., all rights reserved.
```

Comment:
i think this short license relies on the top-level license in the airbytehq/airbyte repo (which doesn't exist here anymore). so i think we need to bring over relevant license stuff to protect it properly.

Reply:
We do actually have this LICENSE https://github.com/airbytehq/airbyte-protocol/blob/main/LICENSE which was copied over from the original `airbyte-protocol` subdir and it only includes the MIT license.  Do we want the Elastic License from the monorepo as well?

Reply:
great! i think you got it right.